src/ColumnSelectionStep/ColumnSelectionStep.tsx

```diff
@@ -164,6 +194,8 @@ export const ColumnSelectionStep: FunctionComponent<ColumnSelectionStepProps> =
 
   return (
     <>
+      {NullsInAidWarning(schema, aidColumn)}
+      <Divider />
```

Comment:
Better to return this divider where it was like Cristian suggested.

Reply:
ok, I'll try to work it out or ask